cabal-install/Main.hs

```diff
@@ -268,6 +270,9 @@ mainWorker args = topHandler $
       , hiddenCmd  win32SelfUpgradeCommand win32SelfUpgradeAction
       , hiddenCmd  actAsSetupCommand actAsSetupAction
       , hiddenCmd  manpageCommand (manpageAction commandSpecs)
+
+      , hiddenCmd  installCommand { commandName = "new-configure" } newConfigureAction
```

Comment:
Right, it's also not install because install changes the user's environment and build does not. By change the user's environment I mean either: adding binaries to user visible path, or adding packages to a user visible environment, like the per-user env.